docs/release-notes/1.7.1.asciidoc

```diff
@@ -0,0 +1,17 @@
+:issue: https://github.com/elastic/cloud-on-k8s/issues/
+:pull: https://github.com/elastic/cloud-on-k8s/pull/
+
+[[release-notes-1.7.1]]
+== {n} version 1.7.1
+
+
+
+
+
```

Comment:
Might want to remove all these empty lines?

Reply:
They are generated automatically, but it does not seem to have an impact on the rendering.